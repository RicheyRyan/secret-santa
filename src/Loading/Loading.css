.Loading {
  text-align: center;
  padding: 2em;
}
