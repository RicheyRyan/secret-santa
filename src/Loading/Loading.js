import React from 'react';
import './Loading.css';

export default props => <div className="Loading">Loading ...</div>;
