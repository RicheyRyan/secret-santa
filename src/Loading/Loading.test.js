import React from 'react';
import renderer from 'react-test-renderer';
import Loading from './Loading.js';

it('Loading component matches snapshot', () => {
  const component = renderer.create(<Loading />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
