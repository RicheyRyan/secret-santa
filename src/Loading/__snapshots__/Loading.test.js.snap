exports[`test Loading component matches snapshot 1`] = `
<div
  className="Loading">
  Loading ...
</div>
`;
